### DISCUSSION TITLE
Install NVM button is not doing anything

### DISCUSSION THREAD

**Discussion by lesaff** [beyondcode/herd-community#492] [Q&A]
I am currently running Herd v1.5.0. I have `nvm` installed separately. The Herd documentation stated that it will use previously installed `nvm` if present. 

When I opened `Node` menu and clicked on the `Install NVM`, it did not do anything. Did I miss a step?

![CleanShot 2024-03-19 at 14 38 51](https://github.com/beyondcode/herd-community/assets/1151181/162f13c5-2967-4c5b-b532-88ac8f5804bc)

**Comment by studiopublica** [comment]
I am having the same issue. Tried uninstalling nvm then reinstall Herd, same issue.

**Comment by JacobBennett** [comment]
Once I installed `nvm` via `brew` I was able to see it in Herd, however before doing so, when clicking on **Install NVM**, got the same thing.

**Reply by lesaff** [reply]
I uninstalled `nvm`, reinstall it and reinstall Herd. Unfortunately, I still cannot see anything in Herd under the Node tab.

**Reply by gavinhewitt** [reply]
> I uninstalled `nvm`, reinstall it and reinstall Herd. Unfortunately, I still cannot see anything in Herd under the Node tab.

Same

**Comment by Mavv3006** [comment]
I have opened #787 for this.
